sql/src/main/java/io/crate/metadata/settings/CrateSettings.java

```diff
@@ -1448,16 +1447,14 @@ public boolean isRuntime() {
                     new SettingsAppliers.IntSettingsApplier(CrateSettings.GATEWAY_RECOVERY_AFTER_NODES))
             .build();
 
-    private static final Set<String> LOGGING_VALUES = ImmutableSet.of("TRACE", "DEBUG", "INFO", "WARN", "ERROR", "FATAL");
-
     /**
      * @return a SettingsApplier for the given setting
      * @throws IllegalArgumentException if the setting isn't supported
      */
     @Nonnull
     public static SettingsApplier getSettingsApplier(String setting) {
         if (isLoggingSetting(setting)) {
-            return new SettingsAppliers.StringSettingsApplier(new StringSetting(setting, LOGGING_VALUES, true));
+            return new SettingsAppliers.LoggingSettingsApplier(setting);
```

Comment:
I would create a new `LoggingSetting` class that inherits from `StringSetting` and overrides the validation setting